projects/audit-logging.md

```diff
@@ -0,0 +1,196 @@
+# Audit Logging
+
+## Background and description
+
+Audit logging describes the capability of capturing audit-trail relevant events of a system to meet compliance requirements. Such events may originate from the infrastructure (e.g. a Kubernetes cluster) up to the application-level. It is a capability that is particularly relevant for providers of enterprise software.
+
+Unlike regular application logs, audit logs are usually subject to long retention periods and software providers must guarantee their completeness (i.e. guarantee of delivery).
+
+Examples of audit logs include: (see [Appendix B: Sample Audit Log Events])
+- failed login attempts
+- permission changes (e.g. of a service account or application user)
+- accessing sensitive information
+- modification of data
+
+### Current challenges
+
+OpenTelemetry does not have a good solution for audit logging
+
+- no semantic conventions for audit logs in OTel
+- OTel APIs/SDKs do not provide feedback to the application level whether data (in particular logs) have been successfully delivered to a remote endpoint. To guarantee delivery, either the SDK has to give those guarantees, or provide feedback to the application so that it can take care of guaranteed delivery itself.
+- OTel collectors may lose audit logs in transit (i.e. no guarantee of delivery)
```

Comment:
Does "OTel collectors" mean the "OpenTelemetry Collector" (https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification#project-naming) or "any collectors that can handle OpenTelemetry data (whether OTLP or something else)"?

Reply:
@reyang This is indeed supposed to refer to the OpenTelemetry Collector. I've rephrased this particular sentence to  "OTel Collector instances may lose audit logs in transit" and adjusted other occurrences of _OTel collectors_ as well. https://github.com/open-telemetry/community/pull/2409/commits/066501b92566ce770eb79f5633d2c1e5166aa9a5